cinema/serializers.py

```diff
@@ -1,24 +1,66 @@
 from rest_framework import serializers
 
-from cinema.models import Movie
+from cinema.models import Movie, Actor, Genre, CinemaHall
 
 
-class MovieSerializer(serializers.Serializer):
+class ActorSerializer(serializers.Serializer):
     id = serializers.IntegerField(read_only=True)
-    title = serializers.CharField(max_length=255)
-    description = serializers.CharField()
-    duration = serializers.IntegerField()
+    first_name = serializers.CharField(max_length=255)
+    last_name = serializers.CharField(max_length=255)
```

Comment:
Consider using `serializers.ModelSerializer` for `ActorSerializer` to simplify the code and automatically handle `create` and `update` methods. This approach reduces redundancy and aligns with Django REST Framework best practices.